static/js/components/dashboard/CourseAction.js

```diff
@@ -81,24 +73,26 @@ export default class CourseAction extends React.Component {
     }
   }
 
-  renderEnrollButton(run: CourseRun): React$Element<*> {
+  renderEnrollButton(run: CourseRun, actionType: string): React$Element<*> {
     let buttonProps = {};
 
     if (this.hasPendingFinancialAid()) {
       buttonProps.disabled = true;
     }
 
     return (
-      <SpinnerButton
-        className="dashboard-button enroll-button"
-        key="1"
-        component={Button}
-        spinning={run.status === STATUS_PENDING_ENROLLMENT}
-        onClick={() => this.handleEnrollButtonClick(run)}
-        {...buttonProps}
-      >
-        Enroll Now
-      </SpinnerButton>
+      <div className="course-action">
```

Comment:
I just retained it because that was what was being rendered before, we could probably just stick the `.course-action` class on the `<button>` directly though